apps/mull-ui-e2e/src/integration/US-1.1.spec.ts

```diff
@@ -38,8 +38,33 @@ frameSizes.forEach((frame) => {
     it('should type into the event description input', () => {
       cy.get('#description').type('test description').should('have.value', 'test description');
     });
-    it('should type into the event location input', () => {
-      cy.get('#location').type('test location').should('have.value', 'test location');
+
+    it('should type into the event location modal and return autocompleted address', () => {
+      cy.get('#location').click();
+      cy.get('#location-input-field').should('be.visible');
+      const res = '845 Rue Sherbrooke';
+      cy.get('#location-input-field').type('845 Rue Sherbrooke');
+      cy.get('#location-input-field-option-0', { timeout: 3500 }).should(
+        'have.text',
+        '845 Rue Sherbrooke, Montreal, QC, Canada'
+      );
```

Comment:
This test fails for me because the first result is 845 Rue Sherbrooke **Ouest**, Montreal, QC, Canada. I'm not sure if this was just not updated from the previous API or it depends on the location where the test is being ran. I suggest the following fix to make it less flaky if it's a location based issue
```suggestion
      cy.get('#location-input-field-option-0', { timeout: 3500 }).should(
        'contain.text',
        '845 Rue Sherbrooke'
      );
```

Reply:
Yeah this most likely wasn't updated

Reply:
Ah I see, my suggestion would make this test API/google update independent so I think we should go with it instead of hardcoding values.

Reply:
Agreed :)